knimin/lib/data_access.py

```diff
@@ -3197,6 +3197,59 @@ def get_property_options(self, prop):
                                prop + '_id'))
             return [dict(x) for x in TRN.execute_fetchindex()]
 
+    def get_or_create_property_option_id(self, prop, option_name):
```

Comment:
I think this function can be used if the user wants to create a property _de novo_ , since it will create it if it doesn't exist and just return the id, I don't think there is any need to have a plate on this function